birdyboard.py

```diff
@@ -0,0 +1,443 @@
+import pickle
+
+
+class Birdyboard:
+
+    def __init__(self):
+        self.users = []
+        self.user_name = None
+
+        self.chirps_library = {"public": [], "private": []}
+        # these variables are assigned when a specific chirp thread is being viewed or added to.
+        self.public_or_private = ""
+        self.chirp_index = None
+
+# ############################################
+# ######## UNLOGGED IN TOP LEVEL MENU ########
+# ############################################
+
+    def unlogged_in_menu_print(self):
+        """
+        prints the choices for an unlogged-in user, at start of app.
+        Arguments: none
+        """
+        print("Welcome. You are not logged in.\n")
+        print("1. new user\n2. log in\n3. view public chirps\n'x' to exit.\n")
+
+    def unlogged_in_menu_next_step(self):
```

Comment:
Since you're calling these both in immediate succession, and the `unlogged_in_menu_print()` method only prints, I'd probably take those prints and just tack them on to the front of the `unlogged_in_menu_next_step()` method, since that's essentially what's happening anyway.